packages/logs-async/logs-async.1.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+name: "logs-async"
+authors: "Vincent Bernardoff <vb@luminar.eu.org>"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+homepage: "https://github.com/vbmithr/logs-async"
+bug-reports: "https://github.com/vbmithr/logs-async/issues"
+dev-repo: "git+https://github.com/vbmithr/logs-async"
+doc: "https://vbmithr.github.io/logs-async/doc"
+build: [ "dune" "build" "-j" jobs "-p" name "@install" ]
```

Comment:
note that `"@install"` is unnecessary as it is already the default target of `dune build`